sound/pci/hda/patch_hdmi.c

```diff
@@ -2833,9 +2833,12 @@ static int alloc_intel_hdmi(struct hda_codec *codec)
 /* parse and post-process for Intel codecs */
 static int parse_intel_hdmi(struct hda_codec *codec)
 {
-	int err;
+	int err, retries = 3;
+
+	do {
+		err = hdmi_parse_codec(codec);
+	} while (err < 0 && retries--);
```

Comment:
@kv2019i you realise, that with this you actually retry up to 4 times, right? ;-) Not a huge deal, so, not sending to the ML

Reply:
@lyakh Ack. Well, one call and three retries :) Anyways, it's a fairly random number that is bigger than 1, so 3 or 4, both are fine.